ui/src/app/pages/homepage/data-access-requirements.tsx

```diff
@@ -293,10 +293,29 @@ const selfBypass = async(spinnerProps: WithSpinnerOverlayProps, reload: Function
   reload();
 };
 
-const Refresh = () => <Button
+const syncExternalModules = async() => {
+  const aborter = new AbortController();
+  try {
+    await Promise.all([
+      profileApi().syncTwoFactorAuthStatus({signal: aborter.signal}),
+      profileApi().syncComplianceTrainingStatus({signal: aborter.signal}),
+      profileApi().syncEraCommonsStatus({signal: aborter.signal}),
```

Comment:
No need to address in this PR, but something to consider: we already drop sync eRA commons cron job because it is useless to AoU(as mentioned in some docs, Terra does not remove user once linked). We might be able to remove this call, and treat eRA linkage is immutable and just one time setup(like login.gov).

Reply:
The purpose of this is to help with the initial setup process.

Reply:
what is initial setup? I thought that is `handleTerraShibbolethCallback`. Then we make a getProfileAPI call, that will return correct user profile state.